src/test_utils.rs

```diff
@@ -64,6 +72,62 @@ impl TestContextManager {
             .build()
             .await
     }
+
+    pub async fn fiona(&mut self) -> TestContext {
+        TestContext::builder()
+            .configure_fiona()
+            .with_log_sink(self.log_tx.clone())
+            .build()
+            .await
+    }
+
+    /// Writes info events to the log that mark a section, e.g.:
+    ///
+    /// ========== `msg` goes here ==========
+    pub fn sec(&self, msg: &str) {
+        self.log_tx
+            .try_send(Event {
+                id: SEC_EVENT_ID,
+                typ: EventType::Info(msg.to_string()),
+            })
+            .expect("The events channel should be unbounded and not closed, so try_send() shouldn't fail");
+    }
+
+    /// - Let one TestContext send a message
+    /// - Let the other TestContext receive it and accept the chat
+    /// - Assert that the message arrived
+    pub async fn send_recv_accept(&self, from: &TestContext, to: &TestContext, msg: &str) {
+        self.sec(&format!(
+            "{} sends a message '{}' to {}",
+            from.name(),
+            msg,
+            to.name()
+        ));
```

Comment:
In the python tests, we only write these section logs in the test itself, not in helper functions - I found them useful here in the helper functions, but of course the question is whether we want them here, doing stuff differently than the Python tests.